raiden/tests/unit/test_transport.py

```diff
@@ -74,14 +93,16 @@ def test_throttle_policy_ping(monkeypatch, raiden_network):
     # Each token corresponds to a single message, the initial capacity is 2
     # meaning the first burst is of 2 packets.
     events[1].wait()
-    check.elapsed(seconds=0, tolerance=tolerance)
 
     assert len(messages) == 4  # two Pings and the corresponding Acks
 
     # Now check the fill_rate for the remaining packets
     for i in range(2, 10):
         events[i].wait()
-        check.elapsed(token_refill, tolerance)
+        test_time.progress(token_refill)
+
+    assert all(t == 0.0 for t in sleeps.sleeptimes[:4]), sleeps.sleeptimes
```

Comment:
Why the slice uses `4`? Does `sleeps.sleeptimes` include an additional sleep per *sent* Ping? Perhaps for the Ack?